### PR TITLE
Register Divergences: v0.0.3

### PR DESCRIPTION
**Pull request by gragusa**
`Divergences` is a Julia package that makes it easy to evaluate divergence measures between two vectors. The package allows calculating the *gradient*  and the diagonal of the *Hessian* of several divergences.

### CODE REVIEW

File: Divergences/versions/0.0.3/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.3
```
Comment (tkelman) [inline Divergences/versions/0.0.3/requires]: this package uses `eachindex`, it doesn't support 0.3: https://travis-ci.org/gragusa/Divergences.jl/jobs/130739315

**Comment by gragusa** [comment]
I retagged v0.0.3 with the right Julia version requirement and manually update the sha-1 of this PR. Is it fine now?

**Comment by tkelman** [comment]
you need to change the requires file here in metadata too

**Comment by gragusa** [comment]
Done. (Hopefully, it is enough). 
Don't worry about closing. Github buttons are what they are!

**Pull request merged**